hepcrawl/spiders/elsevier_spider.py

```diff
@@ -739,7 +740,7 @@ def get_journal_and_section(publication):
 
         try:
             # filter after re.split, which may return empty elements:
-            split_pub = filter(None, re.split(r'(\W+)', publication))
+            split_pub = list(filter(None, re.split(r'(\W+)', publication)))
```

Comment:
i would transform this into a list comprehension, I find `filter(None,...)` very non-obvious:
```suggestion
            split_pub = [element in re.split(r'(\W+)', publication)) if element]
```